roles/agent/.circleci/config.yml

```diff
@@ -0,0 +1,299 @@
+---
```

Comment:
Could we remove the files outside of `defaults/`, `tasks/`, `templates/` and make sure they don't sync? I think they're redundant to have in this repository.

Reply:
Sure  👍  I'll look for a way to do that with github checkout action